pep-0570.rst

```diff
@@ -514,6 +514,36 @@ and for ``varargslist`` would be::
       | '**' vfpdef [',']
     )
 
+--------------------
+Semantic Corner Case
+--------------------
+
+The following is an interesting corollary of the specification.
+Consider this function definition::
+
+    def foo(name, **kwds):
+        return 'name' in kwds
+
+There is no possible call that will make it return ``True``.
+For example::
+
+    >>> foo(1, **{'name': 2})
+    Traceback (most recent call last):
+      File "<stdin>", line 1, in <module>
+    TypeError: foo() got multiple values for argument 'name'
+    >>>
```

Comment:
Nit: Should we remove the extra ">>>" at the end?

Reply:
In general I prefer to keep it -- it indicates that this is all the output there is.